src/Components/Home.jsx

```diff
@@ -0,0 +1,32 @@
+import { useNavigate } from "react-router-dom";
+
+const Home = () => {
+  let navigate = useNavigate();
+  const handleLogin = () => {
+    let path = `/articles`;
+    navigate(path);
+  };
+  return (
+    <div id="login">
+      <form action="#!" id="main">
+        <h2>Login to your account</h2>
+
+        <div class="input-parent">
```

Comment:
The word class is causing an error, please resolve all warnings/errors in the terminal 👍